source/deploy/server/preparations.rst

```diff
@@ -0,0 +1,183 @@
+
+.. important::
+
+    PostgreSQL v13+ is required for Mattermost server installations. MySQL support is being deprecated starting with Mattermost v11.
+
+
+1. Create an PostgreSQL server instance:
+
+   .. tab:: AWS
+   
+      .. code-block:: sh
+
+         sudo apt update
+         sudo apt install postgresql
+    
+   .. tab:: Azure
+   
+      .. code-block:: sh
+
+         sudo apt update
+         sudo apt install postgresql
+    
+   .. tab:: Ubuntu/Debian
+   
+      .. code-block:: sh
+
+         sudo apt update
+         sudo apt install postgresql
+
+   .. tab:: Red Hat/CentOS
+   
+      .. code-block:: sh
+
+         sudo dnf install postgresql-server
+         sudo postgresql-setup --initdb
+
+2. Create the Mattermost database and user:
+
+   a. Access PostgreSQL by running:
+
+      .. code-block:: sh
+
+         sudo -u postgres psql
+
+   b. Create the database:
+
+      .. code-block:: sql
+
+         CREATE DATABASE mattermost;
+
+   c. Create the Mattermost user with a secure password:
+
+      .. code-block:: sql
+
+         CREATE USER mmuser WITH PASSWORD 'mmuser-password';
+
+   d. Grant database access to the user:
+
+      .. code-block:: sql
+
+         GRANT ALL PRIVILEGES ON DATABASE mattermost to mmuser;
+
+   e. If using PostgreSQL v15.x or later, additional grants are required:
+
+      .. code-block:: sql
+
+         ALTER DATABASE mattermost OWNER TO mmuser;
+         GRANT USAGE, CREATE ON SCHEMA PUBLIC TO mmuser;
+
+3. Configure PostgreSQL for remote connections (if database is on a separate server):
+
+   a. Edit postgresql.conf to allow remote connections:
+      
+      .. tab:: Ubuntu/Debian
+      
+         Edit ``/etc/postgresql/{version}/main/postgresql.conf``:
+         
+         .. code-block:: text
+
+            listen_addresses = '*'
+
+      .. tab:: Red Hat/CentOS
+      
+         Edit ``/var/lib/pgsql/{version}/data/postgresql.conf``:
+         
+         .. code-block:: text
+
+            listen_addresses = '*'
+
+   b. Configure client authentication by editing pg_hba.conf:
+      
+      Add the following line, replacing {mattermost-server-IP}:
+      
+      .. code-block:: text
+
+         host all all {mattermost-server-IP}/32 md5
+
+File storage preparation
+-------------------------
+
+Mattermost requires a file storage system for storing user files, images, and attachments. You have several options, including:
+
+- local file storage
+- S3-compatibile object storage (recommended)
+
+Local file storage
+~~~~~~~~~~~~~~~~~~
+
+For simple deployments, you can use local file storage. However, this is not recommended for production environments or multi-node deployments.
+
+1. Create a directory for file storage:
+
+   .. code-block:: sh
+
+      sudo mkdir -p /opt/mattermost/data
+
+2. Set appropriate permissions:
+
+   .. code-block:: sh
+
+      sudo chown -R mattermost:mattermost /opt/mattermost/data
```

Comment:
Did the admin already create the `mattermost` account by this point in the guide?